shell/platform/android/platform_view_android_jni_impl.cc

```diff
@@ -1185,6 +1185,9 @@ bool PlatformViewAndroid::Register(JNIEnv* env) {
     return false;
   }
 
+  // Ensure we don't have any pending exceptions.
+  FML_CHECK(fml::jni::CheckException(env));
```

Comment:
Perhaps capture the result outside the assertion and DCHECK the result instead of a release assertion? Don't want us to crash if there is a pending exception from a non-Flutter component.